### PR TITLE
Added AXtension connector artifacts

### PR DESCRIPTION
**Pull request by JoepKillaars**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: certified-connectors/AXtension Content Gate/readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## AXtension Content Gate Connector
+AXtension Content Gate helps you find all relevant content within a split second, regardless of storage location. Connect your Microsoft Dynamics 365 environment with AXtension� Content Gate and create a single point-of-truth for all content in your organization.
```
Comment (dougelliott) [inline certified-connectors/AXtension Content Gate/readme.md:2]: Looks like there's an issue with the character here. That should be resolved before merging.

Comment (JoepKillaars) [reply certified-connectors/AXtension Content Gate/readme.md]: Removed the invalid character.

File: certified-connectors/AXtension Content Gate/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+              "type": "object",
+              "properties": {
+                "schema": {
+                  "type": "object"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Content Entity properties schema.",
```
Comment (sriyen-msft) [inline certified-connectors/AXtension Content Gate/apiDefinition.swagger.json:347]: @JoepKillaars Please remove the period at the end of the summary value.
Refer - [Documenation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#submission-requirements)
`operation descriptions should be full, descriptive sentences, and end in punctuatio`

Comment (JoepKillaars) [reply certified-connectors/AXtension Content Gate/apiDefinition.swagger.json]: Removed the period.

**Pull request merged**